src/test/java/org/broadinstitute/hellbender/utils/dataflow/DataflowUtilsUnitTest.java

```diff
@@ -49,7 +50,7 @@ public void testThrowException() {
     @Test
     public void testReadFromFileFn(){
         List<SimpleInterval> intervals = Lists.newArrayList(new SimpleInterval("chr7:1-202"), new SimpleInterval("chr8:2-202"));
-        DoFn<File, Read> readfn = new DataflowUtils.LoadReadsFromFileFn(intervals);
+        DoFn<File, Read> readfn = new DataflowUtils.LoadReadsFromFileFn(intervals, ValidationStringency.SILENT);
```

Comment:
Is `SILENT` the right default stringency?  I think lenient might be better to default to since we'll at least notice that it's skipping reads.